packages/lesswrong/lib/collections/posts/helpers.ts

```diff
@@ -200,3 +201,62 @@ export const postGetKarma = (post: PostsBase|DbPost): number => {
 export const postCanEditHideCommentKarma = (user: UsersCurrent|DbUser|null, post?: PostsBase|DbPost|null): boolean => {
   return !!(user?.showHideKarmaOption && (!post || !postGetCommentCount(post)))
 }
+
+/**
+ * Returns the event datetimes in a user-friendly format,
+ * ex: Mon, Jan 3 at 4:30 - 5:30 PM
+ * 
+ * @param {(PostsBase|DbPost)} post - The event to be checked.
+ * @returns {string} The formatted event datetimes.
+ */
+export const prettyEventDateTimes = (post: PostsBase|DbPost): string => {
+  // when no start time, just show "TBD"
+  let eventTime = 'TBD';
+
+  if (post.startTime) {
+    let start = moment(post.startTime)
+    let end = post.endTime && moment(post.endTime)
+    // if we have event times in the local timezone, use those instead
+    const useLocalTimes = post.localStartTime && (!post.endTime || post.localEndTime)
+    if (useLocalTimes) {
+      start = moment(post.localStartTime).utc()
```

Comment:
Timezones are hard, and I trust that you thought about them, but why do we use moment.utc for local times and not for non-local times?

Reply:
Yeah maybe I should explain this here. It has to do with how we are computing the local times. From `postResolvers.ts`:

```
// Compute a denormalized start/end time for events, accounting for the
// timezone the event's location is in. This is subtly wrong: it computes a
// correct timestamp, but then the timezone part of that timezone gets lost
// on the way in/out of the database, so if you use this field, what you're
// getting is "local time mislabeled as UTC".
```